packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -1026,6 +1021,25 @@ describe('<TreeItem />', () => {
           expect(getByTestId('one')).to.have.attribute('aria-selected', 'true');
         });
 
+        it('should deselect a node when space is pressed on a selected node', () => {
```

Comment:
Same as https://github.com/mui-org/material-ui/pull/21901/files#r463968990

Reply:
This would conflict with: 

> Changes keyboard selection to the recommended model (multi-selection without holding modifier keys - https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView)

Reply:
I can't find this message

Reply:
I haven't looked at multi-selection, but isn't it the same?

Reply:
Looking at the https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView, 

For single selection

- I see no mention of the unselect click behavior, I think that the modifier approach would provide the best tradeoff.

For multi-selection

- The "Recommended selection model" seems to only account for keyboard
- The "Alternative selection model" is what macOS implements, I more used to it, but it's not the recommended approach. I don't know what Windows implements